src/frontend/tsx/Challenges.tsx

```diff
@@ -201,10 +206,34 @@ export function Challenges () {
               type="text"
               placeholder={'~sampel-palnet'}
               value={who}
-              onChange={(e) => setWho(e.target.value)}
+              onChange={(e) => {
+                  setWho(e.target.value);
+                  setNewOpp(e.target.value);
+              }}
               key={badChallengeAttempts}
               disabled={ challengingFriend }/>
           </div>
+          <div
+            className="new-opp-tally-container"
+            style={
+              newOpp === ''
+              ? {display: 'none'}
+              : !ob.isValidPatp(`${newOpp}`)
+              ? {display: 'none'}
+              : {display: 'block'}
+            }
```

Comment:
I don't think the linter throws errors about this, but the indentation should be:
```
newOpp === ''
  ? {display: 'none'}
  : !ob.isValidPatp(`${newOpp}`)
    ? {display: 'none'}
    : {display: 'block'}
```